instance/tasks.py

```diff
@@ -92,16 +91,28 @@ def spawn_appserver(
 def make_appserver_active(appserver_id, active=True, deactivate_others=False):
     """
     Mark an AppServer as active or inactive.
+
+    :returns True if successful, False otherwise.
     """
     appserver = OpenEdXAppServer.objects.get(pk=appserver_id)
-    logger.info('%s %s: ID=%s', "Activating" if active else "Deactivating", appserver, appserver_id)
-    appserver.make_active(active=active)
-    if active and deactivate_others:
-        other_appservers = appserver.instance.appserver_set.filter(_is_active=True).exclude(pk=appserver_id)
-        for appserver_to_deactivate in other_appservers:
-            logger.info('Deactivating %s [%s]', appserver_to_deactivate, appserver_to_deactivate.id)
-            appserver_to_deactivate.make_active(active=False)
+    vmserver_ready = appserver.server.status == Server.Status.Ready
+
```

Comment:
@UmanShahzad , this is the new code. you are looking at this ? I am a bit confused. We do want to deactivate without checking the vm server, right?